.rubocop.yml

```diff
@@ -2,5 +2,6 @@
 # Managed by modulesync - DO NOT EDIT
 # https://voxpupuli.org/docs/updating-files-managed-with-modulesync/
 
+inherit_from: .rubocop_todo.yml
```

Comment:
I don't think we need the rubocop_todo file if it's empty?

Reply:
True, we basically have two options. Remove the inherit part on every modulesync pr from now on or just leave an empty rubocop_todo.yml stump.
I would prefer the later, but I am fine with both.

Reply:
we only add the inherit if a rubocop_todo.yml is present. If the file isn't needed anymore, we should remove the rubocop_todo.yml file and cleanup the .rubocop.yml.